pkgs/tools/typesetting/tex/texlive-new/combine.nix

```diff
@@ -158,7 +158,8 @@ in buildEnv {
     texlinks.sh "$out/bin" && wrapBin
     perl `type -P fmtutil.pl` --sys --refresh | grep '^fmtutil' # too verbose
     #texlinks.sh "$out/bin" && wrapBin # do we need to regenerate format links?
-    perl `type -P updmap.pl` --sys --syncwithtrees --force
+    echo "y" | perl `type -P updmap.pl` --sys --syncwithtrees --force
+    perl `type -P updmap.pl` --sys --force
```

Comment:
By reading the output of `nix-rebuild`, I have found some hints.
```
updmap will read the following updmap.cfg files (in precedence order):
```
This means that no `updmap.cfg` could be found. So do not remove it.
```
updmap may write changes to the following updmap.cfg file:
  /nix/store/texmf-config/web2c/updmap.cfg
```
The generated `updmap.cfg` is going to the `texmf-config` subdirectory. So keep it.
```
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2078.
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
updmap: open() failed: No such file or directory at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2078.
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
updmap: open() failed: No such file or directory at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2078.
Use of uninitialized value $fn in concatenation (.) or string at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
updmap: open() failed: No such file or directory at /nix/store/gywb923q8p2kss1jj80l28zna42wl0sc-texlive-combined-2015/share/texmf/scripts/texlive/updmap.pl line 2079.
Missing map files found, disabling
	dvips35.map (in builtin)
	pdftex35.map (in builtin)
	ps2pk35.map (in builtin)
in /nix/store/texmf-config/web2c/updmap.cfg
Do you really want to continue (y/N)? answer =n=
```
Some missing map files were found and `updmap` wants to know if we want to continue. If you choose not to continue (the default answer), no map file will be built. We can pipe `y` to `updmap` in order to answer _yes_ to its question.
```
Please fix manually before running updmap(-sys) again!
```
No need to fix manually. On a second run without the `--syncwithtrees` options, `updmap` builds the map files.

At least it worked this way for me (before the latest updates to `combine.nix` I have seen today).